react_main/src/pages/User/User.jsx

```diff
@@ -188,7 +190,13 @@ export function Avatar(props) {
   else if (large) size = "large";
   else size = "";
 
-  if (hasImage && !imageUrl && id) {
+  if (hasImage && !imageUrl && id && avatarId) {
+    if (id === avatarId && !deckProfile) {
+      style.backgroundImage = `url(/uploads/${id}_avatar.jpg?t=${siteInfo.cacheVal})`;
```

Comment:
![image](https://github.com/UltiMafia/Ultimafia/assets/24848927/9566bdf0-2a51-40e9-8912-640f9e0d7e42)
 still seeing this wonky path

Reply:
It's the default view when I click create deck

Reply:
![image](https://github.com/UltiMafia/Ultimafia/assets/24848927/c7f76de8-4858-4e54-acdf-557b4be672af)
![image](https://github.com/UltiMafia/Ultimafia/assets/24848927/e276173f-61ff-4d2a-b0de-61f84c4c9fa1)

flows to case c but is undefined

Reply:
fixed, default avatar empty string